gym_electric_motor/physical_systems/electric_motors/externally_excited_synchronous_motor.py

```diff
@@ -6,22 +6,23 @@
 
 class ExternallyExcitedSynchronousMotor(SynchronousMotor):
     """
-    =====================  ==========  ============= ===========================================
+    =====================  ==========  ============= =================================================
     Motor Parameter        Unit        Default Value Description
-    =====================  ==========  ============= ===========================================
+    =====================  ==========  ============= =================================================
     r_s                    mOhm        15.55         Stator resistance
     r_e                    mOhm        7.2           Excitation resistance
     l_d                    mH          1.66          Direct axis inductance
     l_q                    mH          0.35          Quadrature axis inductance
     l_m                    mH          1.589         Mutual inductance
     l_e                    mH          1.74          Excitation inductance
     p                      1           3             Pole pair number
+    k                      1           65.21         Transfer ratio of number of windings (N_s / N_e)
```

Comment:
Why do we introduce k as a parameter instead of N_s and N_e? Is that a common approach for EESM simulations?